cmd/local/options.go

```diff
@@ -40,13 +32,47 @@ func NewOptions(o *cli.Options) *Options {
 		"",         // version
 		"",         // profile
 		[]string{}, // components
+		[]string{}, // values
+		"",         // componentsFile
 	}
 }
 func (o *Options) Kubeconfig() string {
 	return o.kubeconfig
 }
 
-func componentsFromStrings(list []string) []keb.Components {
+type KymaComponentList struct {
```

Comment:
You can skip this block (line 43-62) and use `data := test.NewComponentList(path)` instead

Reply:
`NewComponentList(t *testing.T, compListFile string)` has testing dependency. I just reused structs and did Unmarshall call directly.